labelbox/data/annotation_types/metrics/base.py

```diff
@@ -0,0 +1,35 @@
+from abc import ABC
+from pydantic import ValidationError, confloat, BaseModel, validator
+from pydantic.error_wrappers import ErrorWrapper
+from typing import Dict, Optional, Any, Union
+
+ConfidenceValue = confloat(ge=0, le=1)
+
+MIN_CONFIDENCE_SCORES = 2
+MAX_CONFIDENCE_SCORES = 15
```

Comment:
how come `[2,15]` and not `[0,1]`?

Reply:
This is the number of scores. Not the range. We are constraining the range with pydantics confloat type. So there is no point of having confidence if you have less than one value. And if you have more than 15 it is too many details and clutters our db.